stable/azuredisk-csi-driver/templates/csi-azuredisk-node.yaml

```diff
@@ -49,7 +49,7 @@ spec:
             - name: registration-dir
               mountPath: /registration
         - name: azuredisk
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          image: "{{ .Values.image.azuredisk.repository }}:{{ .Values.image.azuredisk.tag }}"
           args:
             - "--v=5"
```

Comment:
let us keep the args notation we do everywhere without double quotes
I know I missed it by myself also 😬